config/application.rb

```diff
@@ -44,5 +44,8 @@ class Application < Rails::Application
 
     # Do not swallow errors in after_commit/after_rollback callbacks.
     config.active_record.raise_in_transactional_callbacks = true
+
+    # Load all models
```

Comment:
Technically we're loading all the models in the `models` directory, including those in subfolders recursively. This is different from the default behavior of using subfolders as namespaces (eg. `foo/bar/some_model.rb` is a file for `Foo::Bar::SomeModel`). Let's write something to that effect in the comments to be clear.